drivers/docker/driver.go

```diff
@@ -977,9 +988,19 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 
 	if len(driverConfig.Labels) > 0 {
 		config.Labels = driverConfig.Labels
-		logger.Debug("applied labels on the container", "labels", config.Labels)
 	}
 
+	config.Labels = map[string]string{
+		dockerLabelTaskID:   task.ID,
+		dockerLabelTaskName: task.Name,
+		dockerLabelAllocID:  task.AllocID,
```

Comment:
Were we just going to ship this label by default initially? I think we can ask users to pay the cost of at least 1 label, but I wasn't sure where we landed on more expanded labels.

Reply:
done - kept dockerLabelAllocID and removed others.